lib/api/client-commands/pause.js

```diff
@@ -25,27 +25,38 @@ util.inherits(Pause, EventEmitter);
 
 Pause.prototype.command = function(ms, cb) {
   // If we don't pass the milliseconds, the client will
-  // be suspended indefinitely
+  // be suspended indefinitely, until the user presses some
+  // key in the terminal to resume it.
   if (!ms) {
-    ms = 1000 * 60 * 60 * 6; // 6 hours
     // eslint-disable-next-line
-    console.log('Waiting... Press Ctrl+C to exit.');
-    process.on('SIGINT', (code) => {
-      this.api.end(function () {
-        process.exit(0);
-      });
+    console.log('Waiting... Press any key to resume, or press Ctrl+C to exit.');
+
+    process.stdin.resume();
+    process.stdin.setRawMode(true);
+    process.stdin.once('data', (data) => {
+      const byteArray = [...data];
+      if (byteArray.length > 0 && byteArray[0] === 3) {
+        // Ctrl+C is pressed
+        this.api.end(function () {
+          process.exit(0);
+        });
+      }
+      process.stdin.setRawMode(false);
+      process.stdin.pause();
+
+      this.emit('complete');
```

Comment:
Can use `keypress` event to write this instead:
```
 process.stdin.once('keypress', (str, key) => {
      if (key.ctrl && key.name === 'c') {
        this.api.end(function() {
          process.exit(0);
        });
      }
      process.stdin.setRawMode(false);
      process.stdin.pause();

      this.emit('complete');
    });
```

Reply:
Updated the PR.